lonboard/_geoarrow/parse_wkb.py

```diff
@@ -54,28 +54,28 @@ def parse_wkb_table(table: pa.Table) -> List[pa.Table]:
     if GeometryType.LINEARRING in unique_type_ids:
         raise ValueError("LinearRings not currently supported")
 
-    point_indices = np.where(
-        (type_ids == GeometryType.POINT) | (type_ids == GeometryType.MULTIPOINT)
+    polygon_indices = np.where(
+        (type_ids == GeometryType.POLYGON) | (type_ids == GeometryType.MULTIPOLYGON)
     )[0]
 
     linestring_indices = np.where(
         (type_ids == GeometryType.LINESTRING)
         | (type_ids == GeometryType.MULTILINESTRING)
     )[0]
 
-    polygon_indices = np.where(
-        (type_ids == GeometryType.POLYGON) | (type_ids == GeometryType.MULTIPOLYGON)
+    point_indices = np.where(
+        (type_ids == GeometryType.POINT) | (type_ids == GeometryType.MULTIPOINT)
     )[0]
 
-    if len(point_indices) > 0:
-        point_field, point_arr = construct_geometry_array(
-            shapely_arr[point_indices],
+    if len(polygon_indices) > 0:
```

Comment:
Before this line, can you add a comment saying something like

> We intentionally first check polygons, then line strings, then points, so that the order of generated layers is polygon, then line strings, then points. This ensures that points are rendered on top and polygons are rendered on the bottom.

In basically all my code I order my checks as point, linestring, polygon, multi point, multi linestring, multi polygon, and so I'd easily forget why we do this ordering